src/Symfony/Bundle/TwigBundle/Loader/NativeFilesystemLoader.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle\Loader;
+
+use Twig\Error\LoaderError;
+use Twig\Loader\FilesystemLoader;
+
+class NativeFilesystemLoader extends FilesystemLoader
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function findTemplate($template, $throw = true)
+    {
+        try {
+            return parent::findTemplate($template, $throw);
+        } catch (LoaderError $e) {
+            if ($e instanceof \Twig_Error_Loader && preg_match('/^([^:]*?)(?:Bundle)?:([^:]*+):(.+\.[^\.]+\.[^\.]+)$/', $template, $m) && 0 !== strpos($template, '@')) {
```

Comment:
the `instanceof` will always be true. `Twig\Error\LoaderError` is a class alias of `Twig_Error_Loader`.